prompt-less signing via passphrase file (#1540)
containers/skopeo: docs/skopeo-copy.1.md

Add a signature using that key ID for an image name corresponding to _destination-image_

**--sign-passphrase-file**=_path_

> Passphare to use when signing with the key ID from `--sign-by`. Only the first line will be read. A passphrase stored in a file is of questionable security if other users can read this file. Do not use this option if you can avoid.

Reviewer: `s/if you can avoid it/& it/` ?

(Everywhere)

> Passphare to use when signing with the key ID from `--sign-by`. Only the first line will be read. A passphrase stored in a file is of questionable security if other users can read this file. Do not use this option if you can avoid it.

Reviewer 2: Sorry @vrothberg, you're breaking our "no pronouns" rule for man pages. Plus a tpyo on the passphrase
```suggestion
The passphrase to use when signing with the key ID from `--sign-by`. Only the first line will be read. A passphrase stored in a file is of questionable security if other users can read this file. Do not use this option if at all avoidable.
```
